src/subPages/AlphaFoldModelSubPage/index.js

```diff
@@ -54,6 +55,12 @@ const AlphaFoldModelSubPage = ({ data, description }) => {
           onModelChange={handleModelChange}
           modelId={modelId}
           selections={selectionsInModel}
+          onModelCifChange={(cif) => {
+            const newURL = cif?.length
+              ? cif.replace('-model', '-confidence').replace('.cif', '.json')
+              : null;
+            if (newURL !== confidenceURL) setConfidenceURL(newURL);
+          }}
```

Comment:
Wouldn't it be possible to handle this in `handleProteinChange()`?

Reply:
nope, `onModelChange` gets trigger when the Select has changed but at that point, the request to get the metadata with links from alphafold is only to be started, and I need the Cif from the response, so basically this method is about knowing the first alphafold request has been received